prover/src/lib.rs

```diff
@@ -34,6 +37,8 @@ mod gpu;
 // ================================================================================================
 
 pub use air::{DeserializationError, ExecutionProof, FieldExtension, HashFunction, ProvingOptions};
+#[cfg(all(target_arch = "x86_64", feature = "cuda"))]
+pub use miden_gpu::cuda::get_num_of_gpus;
```

Comment:
I wonder if we should re-export this from under `cuda` namespace. Maybe something like:
```rust
pub mod cuda {
    pub use miden_gpu::cuda::get_num_of_gpus;
}
```

Reply:
I like it. Updated.